Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py

```diff
@@ -2150,6 +2150,39 @@ def test_http_request_proxy_true(self):
             }
             assert m.called is True
 
+    def test_http_request_proxy_without_http_prefix(self):
```

Comment:
I would also add a test where the proxy is not set and make sure the values will not be `http://` (prefix only)

Reply:
done